src/tools/ScenarioMeasurement/Startup/LinuxTraceSession.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ScenarioMeasurement
+{
+    class LinuxTraceSession : ITraceSession
+    {
+        private PerfCollect perfCollect;
+        private Dictionary<TraceSessionManager.KernelKeyword, PerfCollect.KernelKeyword> linKwMapKernel;
+        private Dictionary<TraceSessionManager.ClrKeyword, PerfCollect.ClrKeyword> linKwMapClr;
```

Comment:
We know these are for Linux because they're in the `LinuxTraceSession` and we know it's a map because of the type. In general we do not abbreviate variables like this, and we avoid encoding type information.
```suggestion
        private Dictionary<TraceSessionManager.KernelKeyword, PerfCollect.KernelKeyword> kernelKeywords;
        private Dictionary<TraceSessionManager.ClrKeyword, PerfCollect.ClrKeyword> clrKeywords;
```

Reply:
Yes I struggled with coming up a descriptive but concise name. Good to know!